Shared/Phrases.cs

```diff
@@ -109,7 +108,7 @@ public static class Preferences
 
             private const string PreferenceUpdated = "Your contact preference has been updated";
 
-            public static Activity GetLocation = MessageFactory.Text("Where are you located? (enter City, State/Province, Country)");
+            public static Activity GetLocation = MessageFactory.Text("Where are you located? (enter ZipCode, Country)");

```

Comment:
Very nice that it's that simple! When you tried the zipcodes for India did it work without having to say India? If it works on zipcode alone we should go with that, but if that's not enough for international then we should keep it like this.

Reply:
yes, we need to put in the country else atlas returns multiple entries for a given zipcode.